master/internal/user/postgres_users_intg_test.go

```diff
@@ -412,12 +424,156 @@ func addTestSession() (model.UserID, model.SessionID, string, error) {
 	return user.ID, session.ID, token, nil
 }
 
-func addTestUser(aug *model.AgentUserGroup) (*model.User, error) {
-	var user model.User
-	uid, err := Add(context.TODO(), &model.User{Username: uuid.NewString()}, aug)
+type opts func(*model.User)
+
+func addTestUser(aug *model.AgentUserGroup, opts ...opts) (*model.User, error) {
```

Comment:
also maybe make this function not return an error and just assert no error?

Oh I see this an existing function, then its fine either way